Content.Client/Atmos/Monitor/UI/Widgets/SensorInfo.xaml.cs

```diff
@@ -53,9 +54,9 @@ public SensorInfo(AtmosSensorData data, string address)
             var threshold = data.GasThresholds[gas];
             var gasThresholdControl = new ThresholdControl(Loc.GetString($"air-alarm-ui-thresholds-gas-title", ("gas", $"{gas}")), threshold, AtmosMonitorThresholdType.Gas, gas, 100);
             gasThresholdControl.Margin = new Thickness(20, 2, 2, 2);
-            gasThresholdControl.ThresholdDataChanged += (type, threshold, arg3) =>
+            gasThresholdControl.ThresholdDataChanged += (type, alarmThreshold, arg3) =>
             {
-                OnThresholdUpdate!(_address, type, threshold, arg3);
+                OnThresholdUpdate!(_address, type, alarmThreshold, arg3);
```

Comment:
What is `alarmThreshold` and why did `threshold` get changed to that here?

Reply:
> What is `alarmThreshold` and why did `threshold` get changed to that here?

Rider warned it as "Parameter hides outer local variable with the same name". I suppose that those are lambda expression args, so it got values from the event "ThresholdDataChanged"
So this naming could mislead the person that will read the code

Reply:
I see, I was misled there :')